apps/console-lite/src/app/components/deal-ticket/deal-ticket-estimates.tsx

```diff
@@ -13,29 +14,36 @@ interface DealTicketEstimatesProps {
   fees?: string;
   notionalSize?: string;
   size?: string;
+  slippage?: string;
 }
 
 interface DataTitleProps {
   children: ReactNode;
   quoteName?: string;
 }
 
-const DataTitle = ({ children, quoteName = '' }: DataTitleProps) => (
+export const DataTitle = ({ children, quoteName = '' }: DataTitleProps) => (
   <dt>
     {children}
     {quoteName && <small> ({quoteName})</small>}
   </dt>
 );
 
 interface ValueTooltipProps {
-  value: string;
+  value?: string;
+  children?: ReactNode;
   description: string;
   id?: string;
 }
 
-const ValueTooltipRow = ({ value, description, id }: ValueTooltipProps) => (
+export const ValueTooltipRow = ({
+  value,
```

Comment:
non-blocking: imho instead 
```
value?: string;
children? ReactNode;
```
clearer would be:

`value: string | ReactNode;`

Reply:
It's nicer to use children, because it can best nested and looks cleaner and more legible than having to stuff it into value